clippy_lints/src/arbitrary_source_item_ordering.rs

```diff
@@ -126,7 +126,7 @@ declare_clippy_lint! {
     ///
     /// [cargo-pgo]: https://github.com/Kobzol/cargo-pgo/blob/main/README.md
     ///
-    #[clippy::version = "1.82.0"]
+    #[clippy::version = "1.84.0"]
```

Comment:
As I wrote in #13818 , this lint seems to be available since v1.84.0:

```
$ cargo clippy --version
clippy 0.1.84 (202008a1b8 2024-12-07)

$ cargo clippy --explain arbitrary_source_item_ordering
### What it does

Confirms that items are sorted in source files as per configuration.

### Why restrict this?

Keeping a consistent ordering throughout the codebase helps with working
as a team, and possibly improves maintainability of the codebase. The
...
```